ports/curl/portfile.cmake

```
@@ -39,7 +39,8 @@ vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
     brotli      CURL_BROTLI
     schannel    CMAKE_USE_SCHANNEL
     sectransp   CMAKE_USE_SECTRANSP
-    
+    disable-ldap  DISABLE_LDAP
+
     INVERTED_FEATURES
     non-http HTTP_ONLY
```

```suggestion

    INVERTED_FEATURES
    non-http HTTP_ONLY
    disable-ldap  DISABLE_LDAP
```
The new feature should be added to `INVERTED_FEATURES` list.